File: SchedulingFront-OLD/src/app/common/models/culture.ts
export class Culture {
    cultureId: number;
    name: string;
    value: string;
    flag: string;
    localizationPair: any;
    active: boolean;

    constructor(name: string = null, value: string = null, flag: string = null) {
        this.name = name;
        this.value = value;
        this.flag = flag;
        this.localizationPair = [];
    }
}

File: SchedulingAPI/Localization/Models/Culture.cs
using System.Collections.Generic;

namespace Localization.Models
{
    public class Culture
    {
        public int CultureId { get; set; }
        public string Name { get; set; }
        public string Flag { get; set; }
        public string Value { get; set; }
        public bool Active { get; set; }
        public Dictionary<string, string> LocalizationPair { get; set; }
        //public IEnumerable<TranslateModel> Translate { get; set; } = new List<TranslateModel>();

        public Culture() {  }

        public Culture(string name, string value, string flag)
        {
            Name = name;
            Value = value;
            Flag = flag;
        }

        public Culture Clear()
        {
            return new Culture();
        }
    }
}
